RELEASE.md

```diff
@@ -5,6 +5,12 @@ at [neow3j.io](https://neow3j.io) is up-to-date.
 
 ## Prepare for Source Code Release
 
+> If this release considers a Neo version update (e.g. `3.4.0` &rarr; `3.5.0`): Check whether the [Neo-Express
+> version](https://github.com/neo-project/neo-express/releases) used in the resources of neow3j's `test-tools` module 
```

Comment:
Can you mention, somewhere, the following file?

`test-tools/src/main/resources/test.properties`, specifically the `neoExpressDockerImage` property

Reply:
yes, added in e79a774